src/main/java/com/sonymobile/jenkins/plugins/mq/mqnotifier/pipeline/MQMessageStep.java

```diff
@@ -107,7 +107,7 @@ protected Void run() throws Exception {
             if (config.getEnableVerboseLogging()) {
                 listener.getLogger().println("Posting JSON message to RabbitMQ:\n" + json.toString(2));
             }
-            MQConnection.getInstance().publish(json);
+            MQConnection.getInstance().publish(json, "publishMQMessage");
```

Comment:
A potential follow-up could be to add the routing key as an optional argument to the pipeline step.